crates/cli/src/opts/transaction.rs

```diff
@@ -76,6 +76,16 @@ pub struct TransactionOpts {
     #[arg(long)]
     pub legacy: bool,
 
+    /// Transaction input containing only the input field.
+    /// By default, both the input and data fields will be set.
+    #[arg(long, default_value = "false")]
+    pub input_only_input: bool,
+
+    /// Transaction input containing only the data field.
+    /// By default, both the input and data fields will be set.
+    #[arg(long, default_value = "false")]
+    pub input_only_data: bool,
```

Comment:
I don't really like introducing additional fields for this behaviour, the only reason why we even set both fields is because some nodes are off spec and expect the legacy data field...

Reply:
Totally agree, but now the node refuse the request if we se both fields.

Reply:
Maybe we can only keep `input_only_input` because new nodes only accept the `tx.input`.